tuning/trainercontroller/operations/logcontrol.py

```diff
@@ -0,0 +1,61 @@
+# Third Party
+from transformers import TrainingArguments
+from transformers.utils import logging
+
+# Local
+from .operation import Operation
+
+logger = logging.get_logger(__name__)
+logger.setLevel(level=logging.DEBUG)
+
+VALID_LOG_LEVELS = {
+    "ERROR": logging.ERROR,
+    "WARNING": logging.WARNING,
+    "INFO": logging.INFO,
+    "DEBUG": logging.DEBUG,
+}
```

Comment:
Why are we redefining the levels here instead of using it from the logging library?

Reply:
Addressed.